README.md

```diff
@@ -13,13 +15,47 @@ For full info, docs and examples, see the
 [Meteor MessageFormat home page](http://messageformat.meteor.com/)
 (or install/clone the smart package and run `mrt` in its `website` directory).
 
-In short, you'll get to use the `{{mf}}` helper.  Simple example:
+Installation
+------------
+To install, simply run the following command in your project folder:
 
-```html
+```
+meteor add gadicc:messageformat
+```
+
+You will then need to add an initialization setting in your project. E.g. in `lib/config.js` add the following:
+
+```
+mfPkg.init('en');
+```
+Replace **'en'** with the native language of your project, if appropriate.
+
+Usage
+----
+Once installed, you can start adding the MessageFormat helper to templates:
+
+```
+<p>This string is translatable</p>
+```
+
+...becomes ...
+```
 <p>{{mf 'trans_string' 'This string is translatable'}}</p>
 ```
 
-but much more complex strings are possible, and are useful even if your
+In short, you'll get to use the `{{mf}}` helper. 
+
+Extracting translation strings
+------------------
+After you have added several translation strings to your templates, you will need to extract those strings for translation. To do so:
+
+1. make sure your Meteor app is running (by typing `meteor` in the project directory)
```

Comment:
I don't think the app needs to be running for mf_extract to work. Did this change recently?